src/Uintra20/Content/uintra20/src/app/ui/pages/home/home-page.html

```diff
@@ -1,11 +1,14 @@
-<div class="row-flex">
-  <div class="col-md-8">
-    <ubl-dynamic-component
-      *ngFor="let panel of otherPanels"
-      [data]="panel.get()"
-    ></ubl-dynamic-component>
-  </div>
-  <div class="col-md-4 aside" *ngIf="latestActivities">
-    <ubl-dynamic-component [data]="latestActivities.get()"></ubl-dynamic-component>
+<div class="page-wrapper">
+  <div class="row-flex">
+    <div class="col-md-8">
+      <ubl-dynamic-component
+        *ngFor="let panel of otherPanels"
+        [data]="panel.get()"
+      ></ubl-dynamic-component>
+    </div>
+    <div class="col-md-4 aside" *ngIf="latestActivities">
+      <ubl-dynamic-component [data]="latestActivities.get()"></ubl-dynamic-component>
+    </div>
   </div>
 </div>
+

```

Comment:
maybe add `<div class="page-wrapper"></div>` in app.component.html around `<router-outlet></router-outlet>`